git/essentials/features-iv/README.md

```diff
@@ -0,0 +1,10 @@
+name: Features IV
+
+description: See what git has to offer!

```

Comment:
this is the same description as Features I, II, and III, we should think of something more representative